### PR TITLE
Beta ready for review and release

### PR DESCRIPTION
**Pull request by DevinWalker**
## Description

I've fully tested and demoed the working `develop` branch for DTD and it's ready for code review then release.

### CODE REVIEW

File: src/DoubleTheDonation/SettingsPageContent.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	 * @param $actions
+	 *
+	 * @return array
+	 */
+	public function addSettingsLink( $actions ) {
+		$new_actions = [
+			'settings' => sprintf(
+				'<a href="%1$s">%2$s</a>',
+				admin_url( 'edit.php?post_type=give_forms&page=give-settings&tab=general&section=double-the-donation' ),
+				__( 'Settings', 'give-double-the-donation' )
```
Comment (ravinderk) [inline src/DoubleTheDonation/SettingsPageContent.php:45]: use esc_html__

File: src/DoubleTheDonation/UpdateDonationReceipt.php [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	 * @since 1.0.0
+	 */
+	public function renderLegacyRow( $donation ) {
+
+		$instructions = $this->getInstructions( $donation->ID );
+
+
+		if ( isset($instructions['company_name']) ) { ?>
+			<tr>
+				<td scope="row"><strong><?php esc_html_e( 'Employer', 'give-funds' ); ?></strong></td>
```
Comment (ravinderk) [inline src/DoubleTheDonation/UpdateDonationReceipt.php]: Why we are using `give-funds` as text domain?

File: src/DoubleTheDonation/UpdateDonationReceipt.php [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+				<td><?php echo $instructions['company_name']; ?></td>
+			</tr>
+
+		<?php }
+
+		if ( isset($instructions['next_steps']) ) { ?>
+			<tr>
+				<td scope="row"><strong><?php esc_html_e( 'Employer Matching', 'give-funds' ); ?></strong></td>
```
Comment (ravinderk) [inline src/DoubleTheDonation/UpdateDonationReceipt.php]: Why we are using `give-funds` as text domain?

File: src/DoubleTheDonation/resources/views/admin/notices/give-inactive.php [hunk 4]
```diff
@@ -3,7 +3,7 @@
 <div class="notice notice-error">
 	<p>
 		<strong><?php _e( 'Activation Error:', 'give-double-the-donation' ); ?></strong>
-		<?php _e( 'You must have', 'give-double-the-donation' ); ?> <a href="https://givewp.com" target="_blank">Give</a>
+		<?php _e( 'You must have', 'give-double-the-donation' ); ?> <a href="https://givewp.com" target="_blank">GiveWP</a>
 		<?php printf( __( 'plugin installed and activated for the %s add-on to activate', 'give-double-the-donation' ), GIVE_DTD_NAME ); ?>
```
Comment (ravinderk) [inline src/DoubleTheDonation/resources/views/admin/notices/give-inactive.php:7]: Use `esc_html__`

File: src/DoubleTheDonation/resources/views/admin/notices/give-inactive.php [hunk 5]
```diff
@@ -3,7 +3,7 @@
 <div class="notice notice-error">
 	<p>
 		<strong><?php _e( 'Activation Error:', 'give-double-the-donation' ); ?></strong>
-		<?php _e( 'You must have', 'give-double-the-donation' ); ?> <a href="https://givewp.com" target="_blank">Give</a>
+		<?php _e( 'You must have', 'give-double-the-donation' ); ?> <a href="https://givewp.com" target="_blank">GiveWP</a>
```
Comment (ravinderk) [inline src/DoubleTheDonation/resources/views/admin/notices/give-inactive.php:6]: Use `esc_html_e`

**Review by ravinderk — CHANGES_REQUESTED** [review]
@DevinWalker I added a few suggestions to this pr.

File: src/DoubleTheDonation/UpdateDonationReceipt.php (hunk 3 above)
Comment (DevinWalker) [reply src/DoubleTheDonation/UpdateDonationReceipt.php]: Fixed

File: src/Addon/ActivationBanner.php [hunk 6]
```diff
@@ -17,13 +17,19 @@ class ActivationBanner {
...
+		// Check for Activation banner class.
+		if ( ! class_exists( 'Give_Addon_Activation_Banner' ) ) {
+			include GIVE_PLUGIN_DIR . 'includes/admin/class-addon-activation-banner.php';
+		}
+
 		// Only runs on admin.
 		$args = [
 			'file'              => GIVE_DTD_FILE,
-			'name'              => GIVE_DTD_NAME,
+			'name'              => 'Double the Donation',
```
Comment (kjohnson) [inline src/Addon/ActivationBanner.php:29]: Does this need to be translatable?

**Review by kjohnson — APPROVED** [review]
I think this is good to submit via SVN for approval.

File: src/Addon/ActivationBanner.php (hunk 6 above)
Comment (mathetos) [reply src/Addon/ActivationBanner.php:29]: I say no. It's their brand name. Add-on names I believe could be translated, like Fee Recovery for example. But in this case, that is THEIR brandname and shouldn't be translated.

File: give-double-the-donation.php [hunk 7]
```diff
@@ -5,10 +5,10 @@
...
 
 /**
- * Plugin Name: Give - Double-the-Donation
- * Plugin URI:  https://givewp.com/addons/BOILERPLATE/
- * Description: Easily integrate with the Double-the-Donation employer matching platform.
- * Version:     1.0.0
+ * Plugin Name: Give - Double the Donation
+ * Plugin URI:  https://givewp.com/addons/give-double-the-donation/
+ * Description: Easily integrate with the Double the Donation employer matching platform.
+ * Version:     1.0.0-beta.1
```
Comment (DevinWalker) [inline give-double-the-donation.php]: This needs to be `1.0.0`

Comment (kjohnson) [reply give-double-the-donation.php]: It was my understanding that we could send the beta to SVN/.ORG and then tag a `1.0.0` with any changes to go live (after approval).

**Pull request merged**